### ISSUE TITLE
Extension doesn't work at all

### ISSUE THREAD

**Issue by Syycorax** [alyssaxuu/slashy#14]
There seems to be nothing happening when I click on the command block.
I tried chrome and Firefox, built-in and custom command, neither seems to work.
Maybe something changed on notion's side that breaks the extension?

**Comment by NolanGC** [comment]
Extension worked first time I tried it and then broke forever after that. Very strange behavior. I hope it gets fixed!

**Comment by alyssaxuu** [comment]
Yeah it appears Notion updated something, I'll try to get it working again if I can.

**Comment by PiiXiieeS** [comment]
It seems a change from notion:
`Uncaught TypeError: Cannot read properties of undefined (reading 'style')`
```
    if ($(".temp-section").prev().find(".notion-focusable")[0].style.background == 'rgba(55, 53, 47, 0.08)') {
```

**Comment by garrofederico** [comment]
Same issue for me.

**Comment by Kshitiz1403** [comment]
Same for me.

**Comment by hungpham3112** [comment]
Same for me
